packages/itinerary-body/src/defaults/route-description-footer.tsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+import React, { ReactElement } from "react";
+import * as S from "../styled";
+import { RouteDescriptionFooterProps } from "../types";
+
+export function DefaultRouteDescriptionFooter({
+  onClick = null,
+  waitMinutes
+}: RouteDescriptionFooterProps): ReactElement {
+  return (
+    <S.ArrivalTimeContainer
+      onClick={() => {
+        onClick();
+      }}
+    >
+      Arrives in {waitMinutes} minutes
```

Comment:
I'm a bit nervous about merging unlocalized code -- is there any way to warn that this is trimet specific and have this disabled by default?

Reply:
I was thinking of having a trimetMode prop passed from the trimet side to the ItineraryBody component and if it's true in otp-ui ((which would only be the case for trimet's implementation)) then the thing will be rendered. Good idea?